contracts/eosio.system/eosio.system-regproducer-rc.md

```diff
@@ -1,15 +1,63 @@
 # eosio.system regproducer
 
-This Ricardian contract for the system action *regproducer* is legally binding and can be used in the event of a dispute. 
+I, {{producer}}, hereby nominate myself for consideration as an elected block producer. 
 
-## regproducer
-    ( const account_name producer
-    , const public_key& producer_key
-    , const std::string& url
-    , uint16_t location );
+If {{producer}} is selected to produce blocks by the eosio contract, I will sign blocks with {{producer_key}} and I hereby certify that I will keep this key secret and secure. 
 
-_Intent: create a producer-config and producer-info object for 'producer'_
+If {{producer}} is unable to perform obligations under this contract I will resign my position by setting by resubmitting this contract with
+the null producer key.  
 
-As an authorized party I {{ signer }} wish to register a producer candidate named {{ account_name }} and identified with {{ producer_key }} located at {{ url }} and located at {{ location }}.
+{{producer}} hereby agrees to only use {{producer_key}} to sign messages under the following scenarios:
 
-All disputes arising from this contract shall be resolved in the EOS Core Arbitration Forum. 
+1. proposing a block at the time appointed by the block scheduling algorithm 
+2. pre-confirmation for  block produced by a 3rd party when it is deemed valid 
+3. confirming a block for which {{producer}} has received 2/3+ pre-confirmation messages from other producers
+
+I hereby accept liability for any and all provable damages that result from:
+
+1. signing two different block proposals with the same timestamp with {{producer_key}
+2. signing two different block proposals with the same block number with {{producer_key}
```

Comment:
Missing the ending curly brace for producer key "}"

Reply:
Got it